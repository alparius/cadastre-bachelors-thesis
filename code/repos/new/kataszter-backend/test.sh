#!/bin/bash

rm -f coverage.txt
echo 'mode: atomic' > coverage.txt

for d in $(go list ./... | grep -v testing); do
    go test -v -p 1 -race -covermode=atomic -coverprofile=coverage.out -coverpkg=$(go list ./...| grep -v testing | tr "\n" ",") $d
    if [ -f coverage.out ]; then
        tail -n +2 coverage.out >> coverage.txt || exit 255
        rm coverage.out
    fi
done
