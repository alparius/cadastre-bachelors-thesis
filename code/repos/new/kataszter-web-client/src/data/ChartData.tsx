interface IChartData {
    key: any;
    data: any;
    counter: number;
}

export default IChartData;
