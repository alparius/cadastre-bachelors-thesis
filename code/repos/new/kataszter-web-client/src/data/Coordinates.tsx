interface ICoordinates {
    lat: number;
    lng: number;
}

export let NewCoordinates: ICoordinates = {
    lat: 0,
    lng: 0
};

export default ICoordinates;
