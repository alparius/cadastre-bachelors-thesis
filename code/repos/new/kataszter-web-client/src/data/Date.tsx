interface IDate {
    year: number;
    month: number;
    day: number;
    comment: string;
}

export let NewDate: IDate = {
    year: 0,
    month: 0,
    day: 0,
    comment: ""
};

export default IDate;
