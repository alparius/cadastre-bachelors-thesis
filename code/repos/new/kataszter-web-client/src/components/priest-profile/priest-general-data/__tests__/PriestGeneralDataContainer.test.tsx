/*===================
  PriestGeneralDataContainer.unit.test.tsx
 ===================*/
import React from "react";

import IPriestProfileData from "../../../../data/priest/PriestProfileData";
import { NewTestPriestProfileData } from "../../../../util/test/data/test-data";
import { createComponentWithIntl, createComponentWithIntlAndRouter, mountWithIntl } from "../../../../util/test/test-wrapper";
import PriestGeneralDataContainer from "../PriestGeneralDataContainer";

const handlers = {
    setEditMode: jest.fn(),
    setPriest: jest.fn()
};

const priest: IPriestProfileData = NewTestPriestProfileData;

describe("PriestGeneralDataContainer element", () => {
    afterEach(() => {
        handlers.setEditMode.mockClear();
        handlers.setPriest.mockClear();
    });

    it("should render an error message when the showError is true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestGeneralDataContainer priest={priest} loading={false} showError editMode={false} {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render an error message when the showError is true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestGeneralDataContainer priest={priest} loading={false} showError editMode {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render the grid with priest profile data when priest is provided without aviability to change", () => {
        const container = createComponentWithIntl(
            <PriestGeneralDataContainer priest={priest} loading={false} showError={false} editMode={false} {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render the grid with priest profile data when priest is provided with aviability to change", () => {
        const container = createComponentWithIntl(
            <PriestGeneralDataContainer priest={priest} loading={false} showError={false} editMode {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render the grid with placeholders when the loading is true", () => {
        const container = createComponentWithIntl(
            <PriestGeneralDataContainer priest={priest} loading showError={false} editMode={false} {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render the grid with placeholders when the loading is true", () => {
        const container = createComponentWithIntl(
            <PriestGeneralDataContainer priest={priest} loading showError={false} editMode {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it.each([
        [0, "2", "3"],
        [1, "2", "3"],
        [2, "2", "3"],
        [3, "2", "3"],
        [4, "2", "3"],
        [5, "2", "3"],
        [6, "2", "3"],
        [7, "2", "3"],
        [8, "2", "3"],
        [9, "2", "3"]
    ])(
        "should  call handleChangeOnGeneralData on %d. input. After first change: %s, After second change: %s",
        (inputNr, firstChange, secondChange) => {
            const container = mountWithIntl(<PriestGeneralDataContainer priest={priest} loading={false} showError={false} editMode {...handlers} />);
            const input = container.find("input").at(Number(inputNr));
            input.simulate("change", { target: { value: firstChange } });
            input.simulate("change", { target: { value: secondChange } });
            expect(handlers.setPriest).toHaveBeenCalledTimes(2);
        }
    );
});
