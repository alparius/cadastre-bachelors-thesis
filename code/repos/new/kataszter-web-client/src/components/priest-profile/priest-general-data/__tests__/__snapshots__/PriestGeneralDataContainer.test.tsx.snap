// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PriestGeneralDataContainer element should render an error message when the showError is true 1`] = `
<div
  className="ui container"
  style={
    Object {
      "marginTop": "7em",
    }
  }
>
  <h2
    className="ui header red"
  >
    Could not estabilish server connection.
  </h2>
  <h3
    className="ui header"
  >
    <a
      aria-current="page"
      className="hoverLink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Back to the Home Page
    </a>
  </h3>
</div>
`;

exports[`PriestGeneralDataContainer element should render an error message when the showError is true 2`] = `
<div
  className="ui container"
  style={
    Object {
      "marginTop": "7em",
    }
  }
>
  <h2
    className="ui header red"
  >
    Could not estabilish server connection.
  </h2>
  <h3
    className="ui header"
  >
    <a
      aria-current="page"
      className="hoverLink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Back to the Home Page
    </a>
  </h3>
</div>
`;

exports[`PriestGeneralDataContainer element should render the grid with placeholders when the loading is true 1`] = `
Array [
  <div
    className="ui four column grid"
  >
    <div
      className="column"
    />
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="rectangular image"
        />
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
      </div>
    </div>
    <div
      className="column"
    />
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
  </div>,
]
`;

exports[`PriestGeneralDataContainer element should render the grid with placeholders when the loading is true 2`] = `
Array [
  <div
    className="ui four column grid"
  >
    <div
      className="column"
    />
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="rectangular image"
        />
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
      </div>
    </div>
    <div
      className="column"
    />
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
  </div>,
]
`;

exports[`PriestGeneralDataContainer element should render the grid with priest profile data when priest is provided with aviability to change 1`] = `
<div
  className="ui two column grid"
>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Full name
        :
      </b>
    </div>
    <div
      className="column text"
      name="name"
    >
      <div
        className="field"
      >
        <div
          className="ui input"
        >
          <input
            id=""
            onChange={[Function]}
            type="text"
            value="Janos"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Date of birth
        :
      </b>
    </div>
    <div
      className="column text"
      name="birthDate"
    >
      <form
        className="ui form"
        onSubmit={[Function]}
      >
        <div
          className="inline unstackable fields"
          style={
            Object {
              "marginBottom": "-2px",
            }
          }
        >
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              year
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "6em",
                }
              }
            >
              <input
                id="grey"
                name="year"
                onChange={[Function]}
                type="number"
                value={2000}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              month
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id="grey"
                name="month"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              day
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id="grey"
                name="day"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
        </div>
        <div
          className="field"
        >
          <i
            style={
              Object {
                "color": "gray",
              }
            }
          >
            comment about date (optional)
          </i>
          <div
            className="ui fluid input"
          >
            <input
              id="grey"
              name="comment"
              onChange={[Function]}
              type="text"
              value="2000.06.06"
            />
          </div>
        </div>
      </form>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Country of birth
        :
      </b>
    </div>
    <div
      className="column text"
      name="birthCountry"
    >
      <div
        className="field"
      >
        <div
          className="ui input"
        >
          <input
            id=""
            onChange={[Function]}
            type="text"
            value="Románia"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        County of birth
        :
      </b>
    </div>
    <div
      className="column text"
      name="birthCounty"
    >
      <div
        className="field"
      >
        <div
          className="ui input"
        >
          <input
            id="grey"
            onChange={[Function]}
            type="text"
            value="Erdély"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Town of birth
        :
      </b>
    </div>
    <div
      className="column text"
      name="birthTown"
    >
      <div
        className="field"
      >
        <div
          className="ui input"
        >
          <input
            id=""
            onChange={[Function]}
            type="text"
            value="Zilah"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Subscribed to theology
        :
      </b>
    </div>
    <div
      className="column text"
      name="subscriptionDate"
    >
      <form
        className="ui form"
        onSubmit={[Function]}
      >
        <div
          className="inline unstackable fields"
          style={
            Object {
              "marginBottom": "-2px",
            }
          }
        >
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              year
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "6em",
                }
              }
            >
              <input
                id="grey"
                name="year"
                onChange={[Function]}
                type="number"
                value={2000}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              month
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id="grey"
                name="month"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              day
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id="grey"
                name="day"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
        </div>
        <div
          className="field"
        >
          <i
            style={
              Object {
                "color": "gray",
              }
            }
          >
            comment about date (optional)
          </i>
          <div
            className="ui fluid input"
          >
            <input
              id="grey"
              name="comment"
              onChange={[Function]}
              type="text"
              value="2000.06.06"
            />
          </div>
        </div>
      </form>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Graduated from studies
        :
      </b>
    </div>
    <div
      className="column text"
      name="graduationDate"
    >
      <form
        className="ui form"
        onSubmit={[Function]}
      >
        <div
          className="inline unstackable fields"
          style={
            Object {
              "marginBottom": "-2px",
            }
          }
        >
          <div
            className="field"
          >
            <b
              style={Object {}}
            >
              year
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "6em",
                }
              }
            >
              <input
                id=""
                name="year"
                onChange={[Function]}
                type="number"
                value={2000}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={Object {}}
            >
              month
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id=""
                name="month"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={Object {}}
            >
              day
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id=""
                name="day"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
        </div>
        <div
          className="field"
        >
          <i
            style={
              Object {
                "color": "gray",
              }
            }
          >
            comment about date (optional)
          </i>
          <div
            className="ui fluid input"
          >
            <input
              id="grey"
              name="comment"
              onChange={[Function]}
              type="text"
              value="2000.06.06"
            />
          </div>
        </div>
      </form>
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Consecration date
        :
      </b>
    </div>
    <div
      className="column text"
      name="consecrationDate"
    >
      <form
        className="ui form"
        onSubmit={[Function]}
      >
        <div
          className="inline unstackable fields"
          style={
            Object {
              "marginBottom": "-2px",
            }
          }
        >
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              year
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "6em",
                }
              }
            >
              <input
                id="grey"
                name="year"
                onChange={[Function]}
                type="number"
                value={2000}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              month
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id="grey"
                name="month"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              day
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id="grey"
                name="day"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
        </div>
        <div
          className="field"
        >
          <i
            style={
              Object {
                "color": "gray",
              }
            }
          >
            comment about date (optional)
          </i>
          <div
            className="ui fluid input"
          >
            <input
              id="grey"
              name="comment"
              onChange={[Function]}
              type="text"
              value="2000.06.06"
            />
          </div>
        </div>
      </form>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Consecration place
        :
      </b>
    </div>
    <div
      className="column text"
      name="consecrationPlace"
    >
      <div
        className="field"
      >
        <div
          className="ui input"
        >
          <input
            id=""
            onChange={[Function]}
            type="text"
            value="tesztplace"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Diocese
        :
      </b>
    </div>
    <div
      className="column text"
      name="diocese"
    >
      <div
        className="field"
      >
        <div
          className="ui input"
        >
          <input
            id="grey"
            onChange={[Function]}
            type="text"
            value="Zilahi"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Resigned
        :
      </b>
    </div>
    <div
      className="column text"
      name="resigned"
    >
      <form
        className="ui form"
        onSubmit={[Function]}
      >
        <div
          className="inline unstackable fields"
          style={
            Object {
              "marginBottom": "-2px",
            }
          }
        >
          <div
            className="field"
          >
            <b
              style={Object {}}
            >
              year
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "6em",
                }
              }
            >
              <input
                id=""
                name="year"
                onChange={[Function]}
                type="number"
                value={2000}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={Object {}}
            >
              month
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id=""
                name="month"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={Object {}}
            >
              day
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id=""
                name="day"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
        </div>
        <div
          className="field"
        >
          <i
            style={
              Object {
                "color": "gray",
              }
            }
          >
            comment about date (optional)
          </i>
          <div
            className="ui fluid input"
          >
            <input
              id="grey"
              name="comment"
              onChange={[Function]}
              type="text"
              value="2000.06.06"
            />
          </div>
        </div>
      </form>
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Retirement date
        :
      </b>
    </div>
    <div
      className="column text"
      name="retirementDate"
    >
      <form
        className="ui form"
        onSubmit={[Function]}
      >
        <div
          className="inline unstackable fields"
          style={
            Object {
              "marginBottom": "-2px",
            }
          }
        >
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              year
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "6em",
                }
              }
            >
              <input
                id="grey"
                name="year"
                onChange={[Function]}
                type="number"
                value={2000}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              month
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id="grey"
                name="month"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              day
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id="grey"
                name="day"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
        </div>
        <div
          className="field"
        >
          <i
            style={
              Object {
                "color": "gray",
              }
            }
          >
            comment about date (optional)
          </i>
          <div
            className="ui fluid input"
          >
            <input
              id="grey"
              name="comment"
              onChange={[Function]}
              type="text"
              value="2000.06.06"
            />
          </div>
        </div>
      </form>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Date of death
        :
      </b>
    </div>
    <div
      className="column text"
      name="deathDate"
    >
      <form
        className="ui form"
        onSubmit={[Function]}
      >
        <div
          className="inline unstackable fields"
          style={
            Object {
              "marginBottom": "-2px",
            }
          }
        >
          <div
            className="field"
          >
            <b
              style={Object {}}
            >
              year
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "6em",
                }
              }
            >
              <input
                id=""
                name="year"
                onChange={[Function]}
                type="number"
                value={2000}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={Object {}}
            >
              month
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id=""
                name="month"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
          <div
            className="field"
          >
            <b
              style={Object {}}
            >
              day
            </b>
            <div
              className="ui fluid input"
              style={
                Object {
                  "width": "5em",
                }
              }
            >
              <input
                id=""
                name="day"
                onChange={[Function]}
                type="number"
                value={6}
              />
            </div>
          </div>
        </div>
        <div
          className="field"
        >
          <i
            style={
              Object {
                "color": "gray",
              }
            }
          >
            comment about date (optional)
          </i>
          <div
            className="ui fluid input"
          >
            <input
              id="grey"
              name="comment"
              onChange={[Function]}
              type="text"
              value="2000.06.06"
            />
          </div>
        </div>
      </form>
    </div>
  </div>
</div>
`;

exports[`PriestGeneralDataContainer element should render the grid with priest profile data when priest is provided without aviability to change 1`] = `
<div
  className="ui two column grid"
>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Full name
        :
      </b>
    </div>
    <div
      className="column text"
      name="name"
    >
      Janos
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Date of birth
        :
      </b>
    </div>
    <div
      className="column text"
      name="birthDate"
    >
      <span
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        2000.06.06
      </span>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Country of birth
        :
      </b>
    </div>
    <div
      className="column text"
      name="birthCountry"
    >
      Románia
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        County of birth
        :
      </b>
    </div>
    <div
      className="column text"
      name="birthCounty"
    >
      Erdély
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Town of birth
        :
      </b>
    </div>
    <div
      className="column text"
      name="birthTown"
    >
      Zilah
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Subscribed to theology
        :
      </b>
    </div>
    <div
      className="column text"
      name="subscriptionDate"
    >
      <span
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        2000.06.06
      </span>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Graduated from studies
        :
      </b>
    </div>
    <div
      className="column text"
      name="graduationDate"
    >
      <span
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        2000.06.06
      </span>
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Consecration date
        :
      </b>
    </div>
    <div
      className="column text"
      name="consecrationDate"
    >
      <span
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        2000.06.06
      </span>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Consecration place
        :
      </b>
    </div>
    <div
      className="column text"
      name="consecrationPlace"
    >
      tesztplace
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Diocese
        :
      </b>
    </div>
    <div
      className="column text"
      name="diocese"
    >
      Zilahi
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Resigned
        :
      </b>
    </div>
    <div
      className="column text"
      name="resigned"
    >
      <span
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        2000.06.06
      </span>
    </div>
  </div>
  <div
    className="row"
    id="grey"
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Retirement date
        :
      </b>
    </div>
    <div
      className="column text"
      name="retirementDate"
    >
      <span
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        2000.06.06
      </span>
    </div>
  </div>
  <div
    className="row"
    id=""
  >
    <div
      align="center"
      className="column text"
    >
      <b>
        Date of death
        :
      </b>
    </div>
    <div
      className="column text"
      name="deathDate"
    >
      <span
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        2000.06.06
      </span>
    </div>
  </div>
</div>
`;
